app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -906,8 +861,8 @@ class PipelineSampleReads extends React.Component {
             <div className={cs.pipelineInfo}>
               PIPELINE {version_display} {pipeline_version_blurb}
             </div>
-            <div className={cs.topRow}>
-              <div className={cs.breadcrumbs}>
+            <ViewHeader className={cs.viewHeader}>
```

Comment:
Why do we need a class here? 
What are the differences in report's header style?

Reply:
The difference is in the `margin-top` and `margin-bottom` amounts between the `<ViewHeader>` and the elements above and below. I thought it better to let the client handle that.